packages/editor/src/components/editor-interface/index.js

```diff
@@ -139,7 +140,10 @@ export default function EditorInterface( {
 			content={
 				<>
 					{ ! isDistractionFree && ! isPreviewMode && (
-						<EditorNotices />
+						<>
+							<EditorNotices />
+							<MainElementWarnings />
```

Comment:
I was not sure about this and would welcome suggestions for better places to trigger the check from.